### ISSUE TITLE
mlpKerasDropout parameterized for neural networks deeper than a single hidden layer

### ISSUE THREAD

**Issue by tyegagne** [topepo/caret#819]
Can the model mlpKerasDropout be parameterized for neural networks deeper than a single hidden layer? If so, how to is defined in the size parameter?

 Thank you.

method_mod <- 'mlpKerasDropout'
tuneGrid <- expand.grid(
  size = 10
  ,dropout = 0.2
  ,batch_size = 256
  ,lr = 0.0002
  ,rho = 0.9
  ,decay = 0
  ,activation = "relu"
)

**Comment by topepo** [comment]
It can but you would need to write a [custom method](https://topepo.github.io/caret/using-your-own-model-in-train.html). You can base it on one of the existing ones:

```r
getModelInfo("mlpKerasDropout")[[1]]
```
